### PR TITLE
Add missing configurations

### PR DESCRIPTION
**Pull request by tscurtu**
Adds missing configurations to the Helm charts documentation.
Sorts misplaced configurations alphabetically to allow easy finding.
Fixes alignment, standardization issues, and tpyos.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,66 +59,72 @@ More information about these configurations can be found in the [Deploy Sematic
...
+| `ray.supports_gpus`                                       | `true` if your Ray workers can use GPUs                               | `false`                                 |
+| `rbac.create`                                             | Enables creation of K8S RBAC resources for Sematic server             | `true`                                  |
+| `rbac.manage_ray`                                         | Allows the Sematic server to manage Ray clusters.                     | `false`                                 |
+| `secret.create`                                           | Creates a K8S secret for Postgres URL                                 | `true`                                  |
+| `service_account.annotations`                             | K8S service account annotations for Sematic server                    | `{}`                                    |
+| `service_account.create`                                  | Enables createion of K8S service account for Sematic server           | `true`                                  |
```
Comment (snoshy) [inline README.md]:
```suggestion
| `service_account.create`                                  | Enables creation of K8S service account for Sematic server           | `true`                                  |
```

File: README.md [hunk 2]
```diff
@@ -59,66 +59,72 @@ More information about these configurations can be found in the [Deploy Sematic
...
+| `ingress.force_ssl`                                       | Forces redirects to HTTPS for Sematic Server even when HTTP is used   | `true`                                  |
+| `ingress.hosts`                                           | K8S ingress hosts YAML spec for Sematic server                        | See default values.yml                  |
+| `ingress.sematic_dashboard_url`                           | Public dashboard URL to K8S ingress for Sematic server                | Not set                                 |
+| `ingress.tls`                                             | K8s ingress TLS YAML spec for Sematic server                          | `[]`                                    |
+| `local_storage.enabled`                                   | Enables using local disk storage for Artifacts for the Sematic server | `false`                                 |
+| `local_Storage.local_storage_path`                        | Sets the local storage path to use for Artifacts                      | `~/.sematic/data`                       |
```
Comment (snoshy) [inline README.md]:
```suggestion
| `local_storage.path`                        | Sets the local storage path to use for Artifacts                      | `~/.sematic/data`                       |
```

fixed wrong capitalization and removed redundant `local_storage_`

**Review by snoshy — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (tscurtu) [reply README.md]: Fixed the capitalization. The config name is already released like this, so keeping the duplicated name.

File: README.md (hunk 1 above)
Comment (tscurtu) [reply README.md]: Done. Did more edits as well.

**Review by snoshy — APPROVED** [review]

**Pull request merged**